### PR TITLE
Ubuntu Core for secure embedded Linux copy update

### PR DESCRIPTION
**Pull request by carkod**
## Done

Ubuntu Core for secure embedded Linux copy update

## QA

- Check https://ubuntu-com-11066.demos.haus/core against [copy docs](https://docs.google.com/document/d/1QoM0Yqx4JfDJ2G3OPt5bds-ximG1jASmU_PWX4iff3E/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4725

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/core/index.html [hunk 1]
```diff
@@ -646,11 +649,19 @@ <h4>Manufacturer update control</h4>
...
     </div>
     <div class="col-6">
-      <h4>App store included</h4>
-      <p class='p-heading--five'>Your own app store and software curation</p>
-      <p>Every device built with Ubuntu Core has a secure app store. Use the global store for access to all the standard apps, or curate your own collection.</p>
-      <p><a class="p-button--neutral is-inline" href="/internet-of-things/appstore" style="margin-left: 0;"><span
-            class="p-link--external">See what an app store can do for your things</span></a></p>
+      <h4>Your dedicated app store</h4>
+      <p class='p-heading--five'>Software curation made easy</p>
+      <p>Every device built with Ubuntu Core has a secure app store. Use the global store for access to thousands of free applications. Curate your own software collection with a private, enterprise app store.</p>
```
Comment (minkyngkm) [inline templates/core/index.html:654]:
```suggestion
      <p>Every device built with Ubuntu Core has a secure app store. Use the <a href="http://snapcraft.io" class="p-link--external">global store</a> for access to thousands of free applications. Curate your own software collection with a private, enterprise app store.</p>
```
```suggestion
      <p>Every device built with Ubuntu Core has a secure app store. Use the global store for access to thousands of free applications. Curate your own software collection with a private, enterprise app store.</p>
```

File: templates/core/index.html [hunk 2]
```diff
@@ -706,32 +717,6 @@ <h4>License compliance</h4>
...
-      }}
-    </div>
-  </div>
-
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-
   <div class="row u-vertically-center">
     <div class="col-6 u-hide--small u-align--center">
```
Comment (minkyngkm) [inline templates/core/index.html:721]: Think it would be better to switch the image and text position so texts on the left and image on the right

**Review by minkyngkm — APPROVED** [review]
Looks good @carkod - just one link is missing from the copy doc. 👍

File: templates/core/index.html [hunk 3]
```diff
@@ -610,9 +610,12 @@ <h2>Built for business</h2>
 <section class='p-strip'>
   <div class="row u-vertically-center">
     <div class="col-6">
-      <h4>Manufacturer update control</h4>
+      <h4>OTA update control</h4>
       <p class='p-heading--five'>Decide which updates go to your devices.</p>
-      <p>As a device manufacturer or a snap publisher, you decide which updates are signed, certified and delivered to your devices, when they happen and to which devices.</p>
+      <p>Over the air updates guarantee secure software in any location. As a device manufacturer or a snap publisher, you decide which updates are signed, certified and delivered to your devices.</p>
+      <p>
+        <a href="/core/features/ota-updates">Learn more about OTA update control&nbsp;&rsaquo;</a>
```
Comment (minkyngkm) [inline templates/core/index.html:617]:
```suggestion
        <a href="/core/features/ota-updates">Learn more about OTA updates&nbsp;&rsaquo;</a>
```

File: templates/core/index.html (hunk 2 above)
Comment (carkod) [reply templates/core/index.html:727]: Done

**Review by minkyngkm — APPROVED** [review]
LGTM

**Pull request merged**